container-tools/confd-deb-app/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# ConfD client application container image
+#
+# Start from the debian stretch linux base image.
+FROM debian:9.8
```

Comment:
Use Debian 10 "Buster" instead, as ConfD 7.2 support libcrypto 1.1 if you rebuild crypto